packages/nextjs/utils/scaffold-eth/getMetadata.ts

```diff
@@ -0,0 +1,30 @@
+export const getMetadata = ({
+  title,
+  description,
+  imageRelativePath = "/thumbnail.jpg",
+}: {
+  title: string;
+  description: string;
```

Comment:
Also should we make this optional ? Since in `MetaHeader` component we had some defaults set maybe we can set default here same as `MetaHeader`  ?

Reply:
we kind of have defaults for it , If we don't specify metadata for a page it would fallback to the metadata specified in app/layout.tsx